docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -0,0 +1,127 @@
+*****************************************
+Introduction to the XO Transaction Family
+*****************************************
+
+What is XO?
+===========
+
+*XO* is an example transaction family contained within the Sawtooth SDK. It is an
+implementation of the popular pen-and-paper game *Tic-tac-toe* (otherwise known as 
+*Noughts and Crosses* or *X's and O's*).
+
+*X's and O's*-style games have been played all over the world for many centuries. The
+origin of this type of game is unclear; however, some historians believe that the first 
+version of *Tic-tac-toe* may have originated in ancient Egypt. Others believe that the 
+modern *Tic-tac-toe* is an evolution of a game known as *Terni Lapilli*, which was 
+popular within the Roman Empire. 
+
+*Tic-tac-toe* also has historical significance within the field of Computer Science. In
+1952, it became the basis of one of the first-ever video games when `OXO <https://en.wikipedia.org/wiki/OXO>`_ was developed by Alexander S. Douglas at the University 
+of Cambridge. More information regarding the origins and properties of *Tic-tac-toe* 
+can be found `here <https://en.wikipedia.org/wiki/Tic-tac-toe>`_.
+
+*XO* was chosen as an example transaction family for Sawtooth due to its simplicity,
+global playerbase, and straightforward implementation as a computer program. Though
+there is essentially no business application for *XO* per se, this example transaction
```

Comment:
Nice introduction. Suggest deleting this phrase: " Though
 +there is essentially no business application for *XO* per se, " 
Why? Brevity. Most people will realize that XO has few business applications, and - who knows - maybe it does!